easybuild/easyconfigs/c/CIRI-long/CIRI-long-1.0.2-foss-2020b.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'PythonBundle'
+
+name = 'CIRI-long'
+version = '1.0.2'
+
+homepage = 'https://github.com/bioinfo-biols/CIRI-long/'
+description = "Circular RNA Identification for Long-Reads Nanopore Sequencing Data"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('scikit-learn', '0.23.2'),
+    ('Pysam', '0.16.0.1'),
+    ('minimap2', '2.18'),
+    ('Cython', '0.29.22'),
+    ('SAMtools', '1.11'),
+    ('Biopython', '1.78'),
+    ('pyspoa', '0.0.8'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('mappy', '2.17', {
+        'checksums': ['ed1460efc9c6785df28065b7e93e93c92227f623a181f1a852dca6e6acb1a15f'],
+    }),
+    ('python-Levenshtein', '0.12.1', {
```

Comment:
@zarybnicky We have dedicated easyconfigs for this, we should keep that up?

Reply:
I looked for all deps other than python-Levenshtein :) I'll submit an update PR